abs.yaml

```diff
@@ -0,0 +1,5 @@
+upload_channels:
+  - sfe1ed40
+channels:
```

Comment:
I assume these are temporary ? Otherwise LGTM.

Reply:
Yup! Temporary to make sure it builds with the dependencies I built. They'll get removed before merge.